autoload/black.vim

```diff
@@ -145,6 +146,7 @@ def Black(**kwargs):
     string_normalization=not configs["skip_string_normalization"],
     is_pyi=vim.current.buffer.name.endswith('.pyi'),
     magic_trailing_comma=not configs["skip_magic_trailing_comma"],
+    preview=not configs["preview"],
```

Comment:
This doesn't seem right, `preview` isn't an inverted option like `skip-string-normalization`. 
```suggestion
    preview=configs["preview"],
```

Reply:
You are absolutely right! The `not` is wrong here.